.github/workflows/main.yml

```diff
@@ -390,6 +390,15 @@ jobs:
   build:
     name: Build wasmtime
     runs-on: ${{ matrix.os }}
+
+    # Only run the release build on main/version branches
+    if: |
+      github.event_name == 'push' &&
+      (github.ref == 'refs/heads/main' ||
+        startsWith(github.ref, 'refs/tags/v') ||
+        startsWith(github.ref, 'refs/heads/release-')) &&
```

Comment:
I think that this may not actually fire for `release-*` branches since we already only run for `push` events on the `main` branch only (plus tags). Could this be inverted as "run if not PR or if release branch?"

Reply:
I had totally forgotten about the top-level `on` predicate, thanks for catching this!